src/sentry/tasks/reports.py

```diff
@@ -912,6 +928,7 @@ def deliver_organization_user_report(timestamp, duration, organization_id, user_
 
     if not dry_run:
         message.send()
+        set_tag("email_sent", True)
```

Comment:
It might be useful to slice in the future the percentage of users that are not receiving the email.